lib/route_translator/host_path_consistency_lambdas.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RouteTranslator
+  module HostPathConsistencyLambdas
+    module_function
+
+    def for_locale(locale)
```

Comment:
since for_locale is the only method used outside this module, you could move lambdas and sanitize_locale outside `module_function`.

Take a look at other files for example